client/packages/common/src/authentication/AuthContext.tsx

```diff
@@ -138,6 +144,13 @@ export const AuthProvider: FC<PropsWithChildrenOnly> = ({ children }) => {
     setCookie(newCookie);
   };
 
+  const {
```

Comment:
Moved this to AuthContext for better access to cookie so that the expiry doesn't get overwritten or disappear!